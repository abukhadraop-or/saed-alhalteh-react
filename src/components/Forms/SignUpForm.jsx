import React from 'react';

function SignUpForm() {
  return <h1> SignUp Form</h1>;
}

export default SignUpForm;
