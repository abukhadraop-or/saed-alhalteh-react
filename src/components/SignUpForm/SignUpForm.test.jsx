import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import SignUpForm from '.';

describe('<SignUpForm /> tests', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<SignUpForm />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
