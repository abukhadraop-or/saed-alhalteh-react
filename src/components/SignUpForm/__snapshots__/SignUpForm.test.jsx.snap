// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpForm /> tests matches the snapshot 1`] = `
<h1>
   SignUp Form
</h1>
`;
